generate/generate_hcl_check_test.go

```diff
@@ -369,15 +369,10 @@ func TestCheckOutdatedIgnoresWhenGenHCLConditionIsFalse(t *testing.T) {
 	stackEntry := s.CreateStack("stacks/stack")
 	stack := stackEntry.Load(s.Config())
 
-	changedStacks := false
```

Comment:
I did this to only reload if there were changes, so tests are faster. Does it has any problem?

Reply:
It was to avoid me making mistakes =P